source/databricks/calculation_engine/package/datamigration/migration_scripts/202408290841_create__basis_data_time_series_points.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE IF NOT EXISTS {CATALOG_NAME}.{WHOLESALE_BASIS_DATA_INTERNAL_DATABASE_NAME}.time_series_points
+(
+    calculation_id STRING NOT NULL,
+    metering_point_id STRING NOT NULL,
+    metering_point_type STRING NOT NULL,
+    resolution STRING NOT NULL,
+    grid_area_code STRING NOT NULL,
+    energy_supplier_id STRING,
+    observation_time TIMESTAMP NOT NULL,
+    quantity DECIMAL(18, 3) NOT NULL,
+    quality STRING NOT NULL
+)
+USING DELTA
```

Comment:
Let's add clustering:
```suggestion
CLUSTER BY (calculation_id, grid_area_code, energy_supplier_id, observation_time)
USING DELTA
```

Reply:
Doesn’t work